variables.tf

```diff
@@ -14,20 +19,81 @@ variable "subnet_id" {
 #
 # These parameters have reasonable defaults.
 # ------------------------------------------------------------------------------
-variable "ami_owner_account_id" {
-  default     = "self"
-  description = "The ID of the AWS account that owns the Example AMI, or \"self\" if the AMI is owned by the same account as the provisioner."
+
+variable "aws_region" {
+  default     = "us-east-1"
+  description = "The AWS region where the non-global resources for the Cyber Hygiene account are to be provisioned (e.g. \"us-east-1\")."
   type        = string
 }
 
-variable "aws_availability_zone" {
-  default     = "a"
-  description = "The AWS availability zone to deploy into (e.g. a, b, c, etc.)."
+variable "cyhy_lambda_artifacts_s3_bucket_prefix" {
+  default     = "cool-cyhy-lambda-deployment-artifacts"
+  description = "The prefix of the name of the bucket in the Cyber Hygiene account where any Lambda deployment artifacts for a CyHy environment will be stored.  A unique bucket name beginning with the specified prefix will be created."
   type        = string
 }
 
-variable "aws_region" {
-  default     = "us-east-1"
-  description = "The AWS region to deploy into (e.g. us-east-1)."
+variable "provisionaccount_role_description" {
+  default     = "Allows sufficient permissions to provision all AWS resources in the Cyber Hygiene account."
+  description = "The description to associate with the IAM role that allows sufficient permissions to provision all AWS resources in the Cyber Hygiene account."
   type        = string
 }
+
+variable "provisionaccount_role_name" {
+  default     = "ProvisionAccount"
+  description = "The name to assign the IAM role that allows sufficient permissions to provision all AWS resources in the Cyber Hygiene account."
+  type        = string
+}
+
+variable "provisioncyhyroot_policy_description" {
+  default     = "Allows sufficient permissions to provision all AWS resources required by cisagov/cyhy-tf-root."
+  description = "The description to associate with the IAM policy that allows sufficient permissions to provision all AWS resources required by cisagov/cyhy-tf-root."
+  type        = string
+}
+
+variable "provisioncyhyroot_policy_name" {
+  default     = "ProvisionCyHyRoot"
+  description = "The name to assign the IAM policy that allows sufficient permissions to provision all AWS resources required by cisagov/cyhy-tf-root."
+  type        = string
+}
```

Comment:
I don't think these two variables (added in commit 1abf8f029fe06c1a8dbea7e9ec05530d50022572) are actually used.

Reply:
Doh- I was saving those for an upcoming PR and unintentionally committed them.  I removed them in  https://github.com/cisagov/cool-accounts-cyhy/pull/1/commits/a90d036c77bf027808de0791eb40b993ad07e9f4.

Reply:
Why not just address this and [this](https://github.com/cisagov/cool-accounts-cyhy/pull/1#pullrequestreview-2323123413) by:
1. Squashing commit a90d036 into commit 1abf8f0.
2. Amending the resulting commit to also fixup `README.md`.
Then it'll be like it never happened.

Reply:
I think I cleaned this all up, but please let me know if I missed anything.

Reply:
@jsf9k Can this conversation now be resolved?